### DISCUSSION TITLE
How do you forward your Starship configuration to another host?

### DISCUSSION THREAD

**Discussion by nagromc** [starship/starship#4020] [Q&A]
I love my local Starship configuration. But when I SSH to another host, I’m facing a dull, sober, boring, tasteless prompt.

Also, I often SSH to ephemeral boxes such as test Docker containers, so when I log in, I have a fresh, default configuration.

What would be a good method to forward (dynamically if possible) the configuration to the target host at login time?

**Comment by NetRat88** [comment]
This is an old question, but I am also interessted how I need to set up my remote hosts. I am using now starship with Tabby on my MacOS but I connect very often to my thre raspberry Pi's and two QNAP systems via SSH. Is it really neccessary to config starship, themes and fonts directly at the remote hosts?

Best regards
NetRat

**Comment by Vrishabh-D-B** [comment]
I need to know this too.
